sdk/cosmos/azure-cosmos-spark_3-5_2-12/pom.xml

```diff
@@ -172,6 +180,14 @@
           <groupId>io.netty</groupId>
           <artifactId>netty-all</artifactId>
         </exclusion>
+        <exclusion>
```

Comment:
naive question: how do you decide transitive logging dependencies are to be removed? is it purely during runtime?

Reply:
No by analyzing the maven dependency tree - the problem is that slf4j allows only one implementation. I saw errors because both were laoded (and teh wrong one was chosen) - so, I looked at mvn dpednency tree and fixed this.